src/components/PeriodDimension/PeriodTransfer.js

```diff
@@ -162,35 +177,40 @@ const PeriodTransfer = ({
     return (
         <Transfer
             onChange={({ selected }) => {
-                const formattedItems = selected.map((id) => ({
-                    id,
-                    name: [...allPeriods, ...selectedPeriods].find(
+                const formattedItems = selected.map((id) => {
+                    const matchingItem = [...allPeriods, ...selectedItems].find(
                         (item) => item.id === id
-                    ).name,
-                }))
-                setSelectedPeriods(formattedItems)
+                    )
+
+                    return {
+                        id,
+                        name: matchingItem.name,
+                        isActive: matchingItem.isActive,
+                    }
+                })
                 onSelect(formattedItems)
             }}
-            selected={selectedPeriods.map((period) => period.id)}
+            selected={selectedItems.map((period) => period.id)}
             leftHeader={renderLeftHeader()}
             enableOrderChange
             height={TRANSFER_HEIGHT}
             optionsWidth={TRANSFER_OPTIONS_WIDTH}
             selectedWidth={TRANSFER_SELECTED_WIDTH}
             selectedEmptyComponent={renderEmptySelection()}
-            rightHeader={renderRightHeader()}
+            rightHeader={<RightHeader infoText={infoBoxMessage} />}
```

Comment:
`infoText` vs `infoBoxMessage`, a bit confusing given that they're both internal / invented by us.. I think we should settle on only using one of them as our naming convention. Given that `infoBoxMessage` is exported externally maybe `infoText` has to go?

Reply:
This code was taken from the `ItemSelector`, so the same comments apply there.

Reply:
Updated in both components.